db/cassandra/src/main/java/org/commonjava/indy/cassandra/data/CassandraStoreDataManager.java

```diff
@@ -0,0 +1,595 @@
+package org.commonjava.indy.cassandra.data;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.commonjava.indy.audit.ChangeSummary;
+import org.commonjava.indy.data.IndyDataException;
+import org.commonjava.indy.data.StoreEventDispatcher;
+import org.commonjava.indy.db.common.AbstractStoreDataManager;
+import org.commonjava.indy.db.common.StoreUpdateAction;
+import org.commonjava.indy.infinispan.data.AffectedByStoreCache;
+import org.commonjava.indy.infinispan.data.StoreByPkgCache;
+import org.commonjava.indy.model.core.ArtifactStore;
+import org.commonjava.indy.model.core.Group;
+import org.commonjava.indy.model.core.HostedRepository;
+import org.commonjava.indy.model.core.RemoteRepository;
+import org.commonjava.indy.model.core.StoreKey;
+import org.commonjava.indy.model.core.StoreType;
+import org.commonjava.indy.model.core.io.IndyObjectMapper;
+import org.commonjava.indy.subsys.infinispan.CacheHandle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
+import javax.inject.Inject;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.commonjava.indy.db.common.StoreUpdateAction.DELETE;
+import static org.commonjava.indy.db.common.StoreUpdateAction.STORE;
+import static org.commonjava.indy.model.core.StoreType.group;
+
+@ApplicationScoped
+@Alternative
+public class CassandraStoreDataManager extends AbstractStoreDataManager
+{
+
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
+
+    @Inject
+    CassandraStoreQuery storeQuery;
+
+    @Inject
+    IndyObjectMapper objectMapper;
+
+    @Inject
+    @StoreByPkgCache
+    private CacheHandle<String, Map<StoreType, Set<StoreKey>>> storesByPkg;
```

Comment:
These need to be added to MMENG-715, since they won't be updated consistently when we move to a clustered deployment.

Reply:
Yeah, see. Do we need to update this part in code or we just need to wait the clustered ispn ?

Reply:
I will revert the commit related with this and added them into MMENG-715.